### PR TITLE
fix doc build errors

### PR DESCRIPTION
**Pull request by knorth55**
before
```bash
$ make html
sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v1.8.4
loading translations [en]... 完了
/usr/local/lib/python2.7/dist-packages/requests/__init__.py:83: RequestsDependencyWarning: Old version of cryptography ([1, 2, 3]) may cause slowdown.
  warnings.warn(warning, RequestsDependencyWarning)
保存された環境データを読み込み中...完了
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 22 source files that are out of date
updating environment: [config changed] 262 added, 7 changed, 0 removed
/usr/local/lib/python2.7/dist-packages/recommonmark/parser.py:65: UserWarning: Container node skipped: type=document                                                                                        
  warn("Container node skipped: type={0}".format(mdnode.t))
reading sources... [100%] tutorials/publish_static_image_topic                                                                                                                                              
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/install_softkinetic_camera.rst:42: WARNING: Literal block expected; none found.
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/install_softkinetic_camera.rst:48: WARNING: Literal block expected; none found.
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros/index.rst:175: WARNING: Duplicate substitution definition name: "18c4e2d6".
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros/index.rst:200: WARNING: Duplicate substitution definition name: "656450d8".
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros/index.rst:250: WARNING: Duplicate substitution definition name: "1e18c39d".
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros/nodes/heightmap_converter.md:: WARNING: image file not readable: jsk_pcl_ros/nodes/images/heightmap_converter.png
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros/nodes/organized_edge_detector.rst:50: WARNING: Bullet list ends without a blank line; unexpected unindent.
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros/nodes/spherical_point_cloud_simulator.md:: WARNING: image file not readable: jsk_pcl_ros/nodes/images/spherical_pointcloud_simulator.h
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros_utils/index.rst:10: WARNING: toctree glob pattern u'./*' didn't match any documents
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros_utils/nodes/colorize_height2_d_mapping.md:: WARNING: image file not readable: jsk_pcl_ros_utils/nodes/images/colorize_height_2d_mapping.png
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros_utils/nodes/mask_image_to_depth_considered_mask_image.md:: WARNING: image file not readable: jsk_pcl_ros_utils/nodes/images/mask_image_to_depth_considered_mask_image.png
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros_utils/nodes/planar_point_cloud_simulator.md:: WARNING: image file not readable: jsk_pcl_ros_utils/nodes/images/planar_pointcloud_simulator.h
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros_utils/nodes/plane_concatenator.md:: WARNING: image file not readable: jsk_pcl_ros_utils/nodes/images/plane_concatenator.png
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros_utils/nodes/point_indices_to_mask_image.md:: WARNING: image file not readable: jsk_pcl_ros_utils/nodes/images/point_indices_to_mask_image.png
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros_utils/nodes/pointcloud_to_stl.md:: WARNING: image file not readable: jsk_pcl_ros_utils/nodes/images/pointcloud_to_stl.png
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros_utils/nodes/polygon_array_likelihood_filter.rst:2: WARNING: Title underline too short.

PolygonArrayLikelihoodFilter
=====================
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros_utils/nodes/polygon_points_sampler.md:: WARNING: image file not readable: jsk_pcl_ros_utils/nodes/images/polygon_points_sampler.png
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros_utils/nodes/pose_with_covariance_stamped_to_gaussian_point_cloud.md:: WARNING: image file not readable: jsk_pcl_ros_utils/nodes/images/pose_with_covariance_stamped_to_gaussian_pointcloud.gif
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_perception/nodes/alexnet_object_recognition.rst:14: WARNING: malformed hyperlink target.
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_perception/nodes/alexnet_object_recognition.rst:10: WARNING: Unknown target name: "chainer".
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_perception/nodes/bounding_object_mask_image.rst:8: WARNING: image file not readable: jsk_perception/nodes/./images/bounding_object_mask_image.png
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_perception/nodes/concave_hull_mask_image.rst:2: WARNING: Title underline too short.

ConcaveHullMaskImage
===================
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_perception/nodes/grabcut.md:: WARNING: image file not readable: jsk_perception/nodes/image/grabCut.png
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_perception/nodes/people_pose_estimation_2d.rst:18: WARNING: Literal block expected; none found.
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_perception/nodes/solidity_rag_merge.rst:9: WARNING: Explicit markup ends without a blank line; unexpected unindent.
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_perception/nodes/vgg16_object_recognition.rst:14: WARNING: malformed hyperlink target.
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_perception/nodes/vgg16_object_recognition.rst:10: WARNING: Unknown target name: "chainer".
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_recognition_utils/index.rst:10: WARNING: toctree glob pattern u'./*' didn't match any documents
looking for now-outdated files... none found
pickling environment... 完了
checking consistency... 完了
preparing documents... 完了
writing output... [100%] tutorials/publish_static_image_topic                                                                                                                                               
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros/nodes/capture_stereo_synchronizer.md:50: WARNING: None:any の参照ターゲットが見つかりません: launch/capture_multisense_training_data.launch
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros/nodes/cluster_point_indices_decomposer.md:32: WARNING: None:any の参照ターゲットが見つかりません: ../jsk_perception/nodes/colorize_labels
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros/nodes/fuse_rgb_images.md:5: WARNING: None:any の参照ターゲットが見つかりません: fuse_depth_images.html
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros/nodes/pointcloud_screenpoint.md:21: WARNING: None:any の参照ターゲットが見つかりません: ../../../jsk_pcl_ros/sample/pointcloud_screenpoint_sample.launch
/home/shingo/ros/kinetic/src/jsk-ros-pkg/jsk_recognition/doc/overview.md:38: WARNING: None:any の参照ターゲットが見つかりません: _images/mask_image.png
generating indices... genindex
writing additional pages... search
copying images... [100%] jsk_pcl_ros/nodes/images/organized_edge_detector.png                                                                                                                               
copying downloadable files... [100%] tutorials/code/tutorial_image_publisher.launch                                                                                                                         
copying static files... done
copying extra files... 完了
dumping search index in English (code: en) ... 完了
dumping object inventory... 完了
build 成功, 33 warnings.

The HTML pages are in _build/html.

Build finished. The HTML pages are in _build/html.
```

after
```bash
$ make html
sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v1.8.4
loading translations [en]... 完了
/usr/local/lib/python2.7/dist-packages/requests/__init__.py:83: RequestsDependencyWarning: Old version of cryptography ([1, 2, 3]) may cause slowdown.
  warnings.warn(warning, RequestsDependencyWarning)
保存された環境データを読み込み中...完了
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 22 source files that are out of date
updating environment: [config changed] 262 added, 7 changed, 0 removed
/usr/local/lib/python2.7/dist-packages/recommonmark/parser.py:65: UserWarning: Container node skipped: type=document                                                                                        
  warn("Container node skipped: type={0}".format(mdnode.t))
reading sources... [100%] tutorials/publish_static_image_topic                                                                                                                                              
looking for now-outdated files... none found
pickling environment... 完了
checking consistency... 完了
preparing documents... 完了
writing output... [100%] tutorials/publish_static_image_topic                                                                                                                                               
generating indices... genindex
writing additional pages... search
copying images... [100%] jsk_pcl_ros/nodes/images/organized_edge_detector.png                                                                                                                               
copying downloadable files... [100%] tutorials/code/tutorial_color_filtering.launch                                                                                                                         
copying static files... done
copying extra files... 完了
dumping search index in English (code: en) ... 完了
dumping object inventory... 完了
ビルド 成功.

The HTML pages are in _build/html.

Build finished. The HTML pages are in _build/html.
```

### CODE REVIEW

File: doc/jsk_pcl_ros/nodes/organized_edge_detector.rst [hunk 1]
```diff
@@ -47,7 +47,7 @@ Publishing Topics
 
   Debug image.
   Visualize the input and output image of ``cv::HoughLinesP``.
-.. image:: images/organized_edge_detector_hough_image.png
+  .. image:: images/organized_edge_detector_hough_image.png
```
Comment (YutoUchimi) [inline doc/jsk_pcl_ros/nodes/organized_edge_detector.rst]: Please add a new line before inserting an image.

File: doc/jsk_pcl_ros/nodes/cluster_point_indices_decomposer.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ It also publishes tf of centroids of each cluster and oriented bounding box of t
 * `~label` (`sensor_msgs/Image`):
 
    Label image for each cluster point indices.
-   You can visualize it with [jsk\_perception/ColorizeLabels](../jsk_perception/nodes/colorize_labels.md)
+   You can visualize it with [jsk_perception/ColorizeLabels](../../jsk_perception/nodes/colorize_labels.md)
```
Comment (YutoUchimi) [inline doc/jsk_pcl_ros/nodes/cluster_point_indices_decomposer.md]: `../../jsk_perception/nodes/colorize_labels.md` -> `../../jsk_perception/nodes/colorize_labels.html`

File: doc/jsk_pcl_ros/nodes/fuse_rgb_images.md [hunk 3]
```diff
@@ -3,4 +3,4 @@
 ![](images/fuse_rgb_images.jpg)
 
 Fuse rgb images ignoring black region in each image.
-See [FuseDepthImages](fuse_depth_images.html) for other info.
+See [jsk_pcl_ros/FuseDepthImages](./fuse_depth_images.md) for other info.
```
Comment (YutoUchimi) [inline doc/jsk_pcl_ros/nodes/fuse_rgb_images.md]: `./fuse_depth_images.md` -> `./fuse_depth_images.html`

**Comment by YutoUchimi** [comment]
Thanks for fixing documents!

BTW, below warnings appear when built with `Sphinx==1.3.3` (from `requirements.txt`).
Is it OK to ignore these?

```
$ make html
sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v1.3.3
making output directory...
loading translations [en]... done
loading pickled environment... not yet created
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 262 source files that are out of date
updating environment: 262 added, 0 changed, 0 removed
/home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/venv/local/lib/python2.7/site-packages/recommonmark/parser.py:280: UserWarning: Unsupported inline type Hardbreak                                                                                                                                      
  warn("Unsupported inline type " + inline.t)
reading sources... [100%] tutorials/publish_static_image_topic                                                                                                                                                                                                                                                      
/home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros/index.rst:None: WARNING: duplicate label nodes and nodelets, other instance in /home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/imagesift/index.rst
/home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros/nodes/color_histogram.md:4: WARNING: nonlocal image URI found: https://user-images.githubusercontent.com/1901008/26964249-a3a89382-4d2b-11e7-8ce1-c4a12f57ee06.gif
/home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros/nodes/color_histogram_classifier.md:4: WARNING: nonlocal image URI found: https://user-images.githubusercontent.com/1901008/27892667-fc4ddd9c-623b-11e7-9b7a-9349f3711790.png
None:77: WARNING: nonlocal image URI found: https://user-images.githubusercontent.com/1901008/27870629-4f58019c-61de-11e7-8018-8a4d4ccd7e8d.png
/home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros/nodes/color_histogram_filter.md:4: WARNING: nonlocal image URI found: https://cloud.githubusercontent.com/assets/1901008/26710312/f1343f54-4793-11e7-8848-9a2f31e4b5d5.png
/home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros/nodes/color_histogram_visualizer.md:4: WARNING: nonlocal image URI found: https://user-images.githubusercontent.com/1901008/27892782-924c6ed0-623c-11e7-9219-28a849cfe8e0.png
/home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros/nodes/color_histogram_visualizer.md:4: WARNING: nonlocal image URI found: https://user-images.githubusercontent.com/1901008/26914207-e118a968-4c5a-11e7-862f-705895aeee77.gif
/home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros/nodes/primitive_shape_classifier.md:4: WARNING: nonlocal image URI found: https://user-images.githubusercontent.com/1901008/27854434-ca088eee-61a1-11e7-9a98-0864013a8f63.png
/home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros_utils/index.rst:None: WARNING: duplicate label nodes and nodelets, other instance in /home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros/index.rst
/home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/jsk_perception/index.rst:None: WARNING: duplicate label nodes and nodelets, other instance in /home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros_utils/index.rst
/home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/jsk_perception/nodes/draw_rects.rst:7: WARNING: nonlocal image URI found: https://user-images.githubusercontent.com/1901008/31119884-7063d85e-a86d-11e7-9d33-8b8290de4168.gif
/home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/jsk_perception/nodes/draw_rects.rst:9: WARNING: nonlocal image URI found: https://user-images.githubusercontent.com/1901008/31119889-758b560e-a86d-11e7-9f84-3b9dd8db17bd.png
/home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/jsk_perception/nodes/draw_rects.rst:None: WARNING: nonlocal image URI found: https://user-images.githubusercontent.com/1901008/31119893-7cbcb9c2-a86d-11e7-9f04-ab01462acd65.jpg
/home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/jsk_perception/nodes/face_pose_estimation.rst:9: WARNING: nonlocal image URI found: https://user-images.githubusercontent.com/1901008/31308440-f7ffb6a6-abb1-11e7-8b89-0a332001448b.png
/home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/jsk_perception/nodes/ssd_object_detector.md:4: WARNING: nonlocal image URI found: https://user-images.githubusercontent.com/1901008/28256579-d2351040-6afe-11e7-9d65-d899c3f04e8a.png
/home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/jsk_perception/nodes/ssd_object_detector.md:18: WARNING: nonlocal image URI found: https://user-images.githubusercontent.com/1901008/30326332-35d2e240-9803-11e7-8022-8ae0c3ecf239.jpg
/home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/jsk_recognition_utils/index.rst:3: WARNING: Duplicate explicit target name: "nodes and nodelets".
/home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/resized_image_transport/index.rst:None: WARNING: duplicate label nodes and nodelets, other instance in /home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/jsk_perception/index.rst
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
/home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/jsk_pcl_ros/nodes/color_histogram_matcher.md:: WARNING: Pygments lexer name u'' is not known                                                                                                                                                           
writing output... [100%] tutorials/publish_static_image_topic                                                                                                                                                                                                                                                       
generating indices... genindex
writing additional pages... search
copying images... [100%] jsk_pcl_ros/nodes/images/organized_edge_detector.png                                                                                                                                                                                                                                       
copying downloadable files... [100%] /home/yuto/Projects/test/src/jsk-ros-pkg/jsk_recognition/doc/tutorials/code/tutorial_get_masked_image.launch                                                                                                                                                                   
copying static files... done
copying extra files... done
dumping search index in English (code: en) ... done
dumping object inventory... done
build succeeded, 19 warnings.

Build finished. The HTML pages are in _build/html.
```

File: doc/jsk_pcl_ros/nodes/cluster_point_indices_decomposer.md (hunk 2 above)
Comment (knorth55) [reply doc/jsk_pcl_ros/nodes/cluster_point_indices_decomposer.md]: markdown document links will be compiled to html link.

File: doc/jsk_pcl_ros/nodes/fuse_rgb_images.md (hunk 3 above)
Comment (knorth55) [reply doc/jsk_pcl_ros/nodes/fuse_rgb_images.md]: markdown document links will be compiled to html link.

**Comment by knorth55** [comment]
how about using latest `sphinx`?

**Comment by YutoUchimi** [comment]
Markdown link is compiled with new Sphinx(=1.8.4), and `WARNING: nonlocal image URI found` and `WARNING: duplicate label` don't appear anymore! LGTM!

**Comment by knorth55** [comment]
i made test readthedoc with my own fork branch, and it works
https://knorth55jsk-recognition.readthedocs.io/en/fix-doc/

**Comment by YutoUchimi** [comment]
Travis passed!

**Pull request merged**